src/components/atoms/Header.styles.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import { colors } from './../../constants/colors';
+
+export const HeaderContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 46px;
+
+  @media (min-width: 1440px) {
+    height: 36px;
+    margin: 14px 127px;
+  }
+`;
+
+export const LeftSection = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const MenuButton = styled.button`
+  background: none;
+  border: none;
+  cursor: pointer;
+  width: 24px;
+  height: 24px;
+  margin-top: 11px;
+  margin-bottom: 11px;
+  margin-left: 20px;
```

Comment:
write all the margin in one line

Reply:
Changed all the margins properties to padding